### DISCUSSION TITLE
How to stabilize the image

### DISCUSSION THREAD

**Discussion by renatadeabreua** [ultralytics/yolov5#8155] [Q&A]
Hello!

I'm creating a project to detect cans issues. 

I downloaded the model on the ultralytics app but the correct results are appearing in a very specific camera position. 

When I upload the image on a ultralytics website the results are correct, but using the camera this isn't works.

**Comment by glenn-jocher** [comment]
@renatadeabreua 👋 hi, thanks for letting us know about this possible problem with YOLOv5 🚀. Can you provide screenshots of the problem and steps to help us reproduce this on our side?

**Reply by renatadeabreua** [reply]
Hi Glenn,

Please watch the video below:

https://www.youtube.com/shorts/OmXQ-RaVBVw

In the video we are showing what is happening.

Thanks!!

**Reply by glenn-jocher** [reply]
@renatadeabreua I think you might want to include more variation in your training data, i.e. your deployment space should be similar to your training space for best generalization. If you for example only train with cans in one location the model will only work with cans in that location.

![Screenshot 2021-08-04 at 13 15 05](https://user-images.githubusercontent.com/26833433/174320902-ff90959c-ff4c-494f-9600-820872276aac.jpg)

**Reply by glenn-jocher** [reply]
@renatadeabreua in your TikTok it's hard to see, are there two classes when the phone moves? Does the box stay the same color? It should change color when classes change...
